src/decisionengine_modules/glideinwms/transforms/glidein_requests.py

```diff
@@ -170,8 +167,8 @@ def merge_requests(self, manifests, group_manifests):
             m_keys = set(manifests.keys())
             g_keys = set(group_manifests.keys())
             if m_keys != g_keys:
-                raise RuntimeError(
-                    'Mismatch in manifest keys: %s, %s' % (m_keys, g_keys))
+                self.logger.exception(f"Mismatch in manifest keys: {m_keys}, {g_keys}")
+                raise RuntimeError()
```

Comment:
Here and few lines below the message is all in the logging instruction and nothing in the exception.
Maybe something should be also in the exception that is being propagated up

Reply:
HI Marco. The logging.exception call automatically puts the stack trace into the message, so the exception is logged.